### ISSUE TITLE
Final report

### ISSUE THREAD

**Issue by ramanshah** [ramanshahdatascience/consult#2]
I recently hacked the SOW template to do a quick and not-so-bad final consulting report. I should refactor that work into `consult` and document it adequately with some improvements:

- Fewer metadata elements. 
- `proposal_type` is now too specific a name for the top title.
- Current mechanism for putting a left-justified consultant name/byline in the top right is not working. Now that I'm "Raman Shah Data Science, LLC," the positioning is wrong. Also, a logo is coming soon. I may want to patch a logo graphic into `consult` deliverables very soon. Or maybe not: the lack of binary input assets is appealing for its simplicity.
- I bet my design firm will have words for me in terms of design system.

**Comment by ramanshah** [comment]
Possibility for placement of logo is using the `wallpaper` package:

https://ctan.org/tex-archive/macros/latex/contrib/wallpaper/

**Comment by ramanshah** [comment]
I verified that I've nearly reproduced the March final consulting report with `consult`:

- I had to rename `report_title` -> `doc_title` - this is the one breaking change in the YAML metadata.
- The text marketing name/byline is in an unattractive spot. We're hoping to blow this away with a logo feature going forward.

Residual actions:

- Logo feature.
- Document that `consult` is now for reports, too.
